apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -230,6 +230,113 @@ class SbbolFintechApi extends SbbolRequestApi {
+     *          "type":"calculated",
+     *          "passive":true,
+     *          "openDate":"2000-01-01",
+     *          "closeDate":null,
+     *          "state":"OPEN",
+     *          "mode":"STANDART",
+     *          "dbo":true,
+     *          "business":false,
+     *          "businessNewType":false,
+     *          "notDelay":false,
+     *          "urgent":false,
+     *          "minBalance":null,
+     *          "overdraft":null,
+     *          "comment":null,
+     *          "cdiPermDocQnt":2,
+     *          "cdiPermDocSum":"790.00",
+     *          "cdiAcptDocQnt":null,
+     *          "cdiAcptDocSum":null,
+     *          "cdiCart2DocQnt":null,
+     *          "cdiCart2DocSum":null,
+     *          "debitBlocked":false,
+     *          "debitBlockedCause":null,
+     *          "debitBlockedInitiator":null,
+     *          "debitBlockedTaxAuthorityCode":null,
+     *          "debitBlockedBeginDate":null,
+     *          "debitBlockedEndDate":null,
+     *          "creditBlocked":false,
+     *          "creditBlockedCause":null,
+     *          "creditBlockedInitiator":null,
+     *          "creditBlockedTaxAuthorityCode":null,
+     *          "creditBlockedBeginDate":null,
+     *          "creditBlockedEndDate":null,
+     *          "blockedSums":[
+     *          ],
+     *          "blockedQueuesInfo":[
+     *          ],
+     *          "blockedSumQueuesInfo":[
+     *          ]
+     *       },
+     *  ]
+     */
+
+    /**
+     * Get state of payment request, posted with `postPaymentRequest`
+     *
+     * @return {Promise<{error: any}|{data: ClientAccounts}>}
+     */
+    async getClientAccounts () {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getClientAccountsRequestPath,
+        })
+        const jsonData = JSON.parse(data)
+        if (statusCode === 200) {
+            return { data: jsonData }
+        } else {
+            return { error: jsonData }
+        }
+    }
+
+    get getClientAccountsRequestPath () {
+        return `${this.apiPrefix}/v1/client-info`
+    }
+
+    /**
+     * @typedef StatementTransactions
+     * @property {Array[Account]} accounts - optional
+     * @example
+     *
+     */
+
+    /**
+     * Get state of payment request, posted with `postPaymentRequest`
+     *
+     * @return {Promise<{error: any}|{data: StatementTransactions}>}
+     */
+    async getStatementTransactions (accountNumber, statementDate, page = 1) {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getStatementTransactionsRequestPath,
+            body: {
+                accountNumber,
+                statementDate,
+                page,
+            },
+        })
+        const jsonData = JSON.parse(data)
+        if (statusCode === 200) {
+            return { data: jsonData }
+        } else {
+            return { error: jsonData }
```

Comment:
Maybe return status code too?
```suggestion
            return { error: jsonData, statusCode }
```